woo-razorpay.php

```diff
@@ -411,13 +419,117 @@ public function autoEnableWebhook()
 
         }
 
+        public function pluginInstrumentation()
+        {
+            $trackObject = new TrackPluginInstrumentation($_POST['woocommerce_razorpay_key_id'], $_POST['woocommerce_razorpay_key_secret']);
+
+            if (empty($_POST['woocommerce_razorpay_key_id']) or
```

Comment:
@abdulwahidsharief , move check at the start of function before these gets used anywhere.